code/__DEFINES/projectile_defines.dm

```diff
@@ -18,6 +18,13 @@
 #define CAL_CROSSBOW "bolt"
 #define CAL_ARROW "arrow"
 
+//Mech Caliber Defines
+#define CAL_MECH_MACHINEGUN 	"20mm"
```

Comment:
20mm is already used in guns

Reply:
Now this one is accurate because of shotguns being 20mm barrels, though I still suggest we change shotgun ammo to be measured in gauge instead because not even 12gauge is 20mm (it's 18.5 mm but it's close enough)

Reply:
Rebel was going to change 20mm to 12g to my knowledge on shotguns.